specs/dwritecore/DWriteCore.md

```diff
@@ -0,0 +1,255 @@
+# DWriteCore
+
+DWriteCore is the Project Reunion version of DirectWrite, the text rendering and layout
+component that ships in Windows. DWriteCore implements the same API as DirectWrite,
+with a few additions described in this document. See the [DirectWrite documentation](https://docs.microsoft.com/en-us/windows/win32/api/_directwrite/)
+for more information.
+
+For an application that already uses DirectWrite, switching to DWriteCore requires minimal changes:
+
+ - Add a reference to the Project Reunion package.
+ - Include dwrite_core.h instead of dwrite_3.h.
+ - Link to DWriteCore.lib instead of DWrite.lib.
+
+In return, the application gets the benefits of Project Reunion, namely, access to the newest APIs
+and functionality regardless of what version of Windows your customer is running.
+
+# Background
+
+<!-- TEMPLATE
```

Comment:
Template stuff needs to be deleted. I'll update the template to indicate as such.

Reply:
RECOMMEND: Yes

Reply:
I think it is indicated in the template. I just missed this one.